tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -60,6 +62,17 @@ Test Metric Existence For "Rhods_Aggregate_Availability" On ODS Prometheus
     @{list_values} =    Create List    1    0
     Should Contain    ${list_values}    ${resp.json()["data"]["result"][0]["value"][-1]}
 
+Verify MT-SRE Can Connect To Prometheus Using Port-Forwarding
```

Comment:
I think this test should not be in the 200_metrics.robot file, but I'm not sure of the best place:
- (Option 1) As the test in polarion is in Component: deploy / Subcomponent: installation > Maybe this test should be in a new file tests/Tests/100__deploy/100__installation/103__post_install_monitoring_port_forwarding.robot and move it there also the grafana port forwarding test?
- (Option 2): move the test to a new file tests/Tests/200__monitor_and_manage/230__prometheus/230__prometheus_port_forwarding.robot

I think I prefer option 1) to maintain the same Component/Subcomponent as Polarion. @mwaykole @bdattoma @pablofelix what do you think?

Reply:
We talked about this in the ODS QE Automation chat (for another test but same question) and agreed that Option 1 was better. Could you implement this? Please ping me if you have any questions

Reply:
Updated.